code/modules/cargo/randomstock.dm

```diff
@@ -115,7 +115,7 @@ var/list/global/random_stock_common = list(
 	"cosmetic" = 2.2,
 	"suitcooler" = 1.2,
 	"officechair" = 1.2,
-	"booze" = 3.5,
+	"booze" = 3.7,
```

Comment:
i increased this weighting a bit because the pool of bottles to draw from is now slightly larger. this is a generally good practise for content diversity, larger pools get more weight